### DISCUSSION TITLE
整合包报错

### DISCUSSION THREAD

**Discussion by lazy-11g** [Meloong-Git/PCL#1740] [提问] [answered]
之前玩的好好的的一个整合包，我也没有改模组和其他的游戏文件，然后突然启动崩溃了，有人会看报错日志吗
[错误报告-29-04-2023_8.43.25_pm.zip](https://github.com/Hex-Dragon/PCL2/files/11359200/-29-04-2023_8.43.25_pm.zip)

**Comment by IF-desu** [comment]
尝试使用JDK11等低版本java

**Comment by Pigeon0v0** [comment, marked as answer]
关闭系统 UTF-8 。

Edit：超时关闭

**Comment by IF-desu** [comment]
![image](https://user-images.githubusercontent.com/86994398/235850131-418f6afc-1804-410a-a2bb-ecae46ede4a7.png)
步骤
